sql/core/src/test/resources/sql-tests/results/postgreSQL/window_part3.sql.out

```diff
@@ -71,7 +71,8 @@ insert into datetimes values
 -- !query schema
 struct<>
 -- !query output
-
+org.apache.spark.sql.AnalysisException
```

Comment:
Not related to this PR but it's weird that the data preparation step for a test is broken. We should fix it later.

Reply:
seems this datatimes table was not used at all. I could refine this test case in new PR.